docs/kcMigration/KeycloakMigration.md

```diff
@@ -0,0 +1,50 @@
+---
+description: KeyCloak (RH-SSO) Migration from Pathfinder to BCGov service
+resourceType: Documentation
+title: BC Government KeyCloak (RH-SSO) Service Migration Plan
+---
+
+## KeyCloak Service Migration
+
+Hi team, we are in the process of transforming KeyCloak from a pathfinder service to an organizational BCGov service. For more information on this Single Sign-On service, refer to [here](https://developer.gov.bc.ca/BC-Government-SSO-Service-Definition).
+
+Part of the migration requires you (application teams using this existing SSO) to update the KeyCloak configurations. As for now, there is no immediate actions to migrate. We will inform the community in [RocketChat #sso channel](https://chat.pathfinder.gov.bc.ca/channel/sso) as well as product owners when ready so that you could plan ahead for some sprint time to work on the relevant tasks.
+
+At the moment, we are collecting application team information for each KC realm. Please visit the rocketchat discussion [here](to-be-generated), find your realm and provide the details listed:
+- Project Name: 
+- KC realm ID: 
+- OpenShift project set/s: 
+
+If your team is not on the OpenShift platform, please provide contact information for:
+- Product owner: 
+- Technical lead: 
+
+
+### Migration plan and phases:
```

Comment:
I think here we should identify who are involved in each phases as personas. Perhaps in bold?

Phase 1. __Platform Services and Platform Operations__ will test out

Phase 4. Work with __Pilot Teams__ to..